dnf-docker-test/features/steps/test_behave.py

```diff
@@ -64,6 +65,14 @@ def diff_package_lists(a, b):
         map(_right_decorator, list(sb - sa)))
 
 
+def diff_query_lists(sack_a, sack_b):
+    """ Computes diff between sack_a and sack_b """
+    sa, sb = list(sack_a.query().installed()), list(sack_b.query().installed())
+    rem = [x for x in sa if x not in sb]
+    inst = [x for x in sb if x not in sa]
```

Comment:
I have try following and these approaches doesn't work (it should find 0 and 2 different packages, but it finds 380 and 382):
    sa, sb = set(list(sack_a.query().installed())), set(list(sack_b.query().installed()))
    rem = sa.difference(sb)
    inst = sb.difference(sa)

    sa, sb = set(list(sack_a.query().installed())), set(list(sack_b.query().installed()))
    rem = sa - sb
    inst = sb - sa

    sa, sb = set(sack_a.query().installed()), set(sack_b.query().installed())
    rem = sa.difference(sb)
    inst = sb.difference(sa)